scudcloud/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '1.2.0'
```

Comment:
Thinking about this now... it may be wise to move this into __init__.py.  Will do this for a later release.